aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -677,7 +729,10 @@ module aptos_framework::account {
         };
 
         // Set `OriginatingAddress[new_auth_key] = originating_address`.
-        let new_auth_key = from_bcs::to_address(new_auth_key_vector);
+        assert!(
```

Comment:
Is this a problem? This seems to be changing the rule here a bit where before you can map the same auth key to multiple originating accounts?

Reply:
> Is this a problem?

Yes, the error code here is to provide better feedback rather than an unintuitive abort in the underlying table

> you can map the same auth key to multiple originating accounts

The `OriginatingAddress` table does not allow a one-to-many mapping, it only allows one-to-one

Hence the operation that I am trying to provide better feedback on is:

1. Rotate auth key of account `0xaaa` to use auth key `0xbbb`
1. Rotate auth key of account `0xccc` to use auth key `0xbbb`
1. The second operation fails because you can't insert into the table again, but not it fails with the error code I added instead of an error code in the table module for trying to add to a slot that is already full